### ISSUE TITLE
Kiwi container images are compressed and recompressed without a benefit

### ISSUE THREAD

**Issue by dcermak** [openSUSE/open-build-service#11955]
# Issue Description

When building container images with Kiwi on OBS, kiwi will compress the resulting images as `tar.xz` from the `tar` archive. However, this image is at the end of the build decompressed by obs and the `tar.xz` is completely discarded.

This is imho a huge waste of resources, as `xz` compression is very costly. It would be great to coordinate with kiwi and don't create this `xz` compressed archive when building inside the build service.

cc @schaefi @davidcassany @Conan-Kudo

**Comment by davidcassany** [comment]
Indeed, there is a waste of cycles by compressing with `xz`, just to decompress it and upload it.

From KIWI PoV the image is just compressed as other images are, so no difference here. Specially since OSInside/kiwi#1997, kiwi container images are compressed with the same logic as any other image, compressed at `kiwi result bundle` call. I am not sure why OBS uploads the uncompressed image, probably the registry uploading process adds some additional constraint?

Anway, containers compression in KIWI can be configured within the KIWI configuration file. In fact a simple yaml like (`/etc/kiwi.yml`):

```yaml
container:
  - compress: false
```

causes the skip any kind of compression for containers.

Would it be feasible to add this file in OBS build environment by default?

**Comment by Conan-Kudo** [comment]
> Would it be feasible to add this file in OBS build environment by default?

It's probably possible to update `obs-build` to do that, but it won't fix it for all the older OBS deployments... We should do a belt-and-suspenders approach here and have the configuration and use our existing obsworker detection to do the right thing.
